support/Pipelines/Bbh/Ringdown.py

```diff
@@ -233,52 +240,33 @@ def start_ringdown(
     )
-    logger.info("Rotation: " + str(evaluated_fot_dict["Rotation"]))
-    logger.info("Match time: " + str(match_time))
-    logger.info("Settling timescale: " + str(settling_timescale))
-    logger.info("Lmax: " + str(int(ringdown_ylm_coefs[0][4])))
+    logger.debug("Rotation: " + str(evaluated_fot_dict["Rotation"]))
+    logger.debug("Match time: " + str(match_time))
+    logger.debug("Settling timescale: " + str(settling_timescale))
+    logger.debug("Lmax: " + str(int(ringdown_ylm_coefs[0][4])))
 
     ringdown_params["MatchTime"] = match_time
     ringdown_params["ShapeMapLMax"] = int(ringdown_ylm_coefs[0][4])
     ringdown_params["PathToAhCCoefsH5File"] = path_to_output_h5
     ringdown_params["AhCCoefsSubfilePrefix"] = output_subfile_prefix
-    ringdown_params["Rotation0"] = evaluated_fot_dict["Rotation"][0][0]
-    ringdown_params["Rotation1"] = evaluated_fot_dict["Rotation"][0][1]
-    ringdown_params["Rotation2"] = evaluated_fot_dict["Rotation"][0][2]
-    ringdown_params["Rotation3"] = evaluated_fot_dict["Rotation"][0][3]
-    ringdown_params["dtRotation0"] = evaluated_fot_dict["Rotation"][1][0]
-    ringdown_params["dtRotation1"] = evaluated_fot_dict["Rotation"][1][1]
-    ringdown_params["dtRotation2"] = evaluated_fot_dict["Rotation"][1][2]
-    ringdown_params["dtRotation3"] = evaluated_fot_dict["Rotation"][1][3]
-    ringdown_params["dt2Rotation0"] = evaluated_fot_dict["Rotation"][2][0]
-    ringdown_params["dt2Rotation1"] = evaluated_fot_dict["Rotation"][2][1]
-    ringdown_params["dt2Rotation2"] = evaluated_fot_dict["Rotation"][2][2]
-    ringdown_params["dt2Rotation3"] = evaluated_fot_dict["Rotation"][2][3]
-    ringdown_params["ExpansionOuterBdry"] = evaluated_fot_dict[
-        "ExpansionOuterBoundary"
-    ][0]
-    ringdown_params["dtExpansionOuterBdry"] = evaluated_fot_dict[
-        "ExpansionOuterBoundary"
-    ][1]
-    ringdown_params["dt2ExpansionOuterBdry"] = evaluated_fot_dict[
-        "ExpansionOuterBoundary"
-    ][2]
-    ringdown_params["Translation0"] = evaluated_fot_dict["Translation"][0][0]
-    ringdown_params["Translation1"] = evaluated_fot_dict["Translation"][0][1]
-    ringdown_params["Translation2"] = evaluated_fot_dict["Translation"][0][2]
-    ringdown_params["dtTranslation0"] = evaluated_fot_dict["Translation"][1][0]
-    ringdown_params["dtTranslation1"] = evaluated_fot_dict["Translation"][1][1]
-    ringdown_params["dtTranslation2"] = evaluated_fot_dict["Translation"][1][2]
-    ringdown_params["dt2Translation0"] = evaluated_fot_dict["Translation"][2][0]
-    ringdown_params["dt2Translation1"] = evaluated_fot_dict["Translation"][2][1]
-    ringdown_params["dt2Translation2"] = evaluated_fot_dict["Translation"][2][2]
+    ringdown_params["Rotation"] = str(evaluated_fot_dict["Rotation"][0])
+    ringdown_params["dtRotation"] = str(evaluated_fot_dict["Rotation"][1])
+    ringdown_params["dt2Rotation"] = str(evaluated_fot_dict["Rotation"][2])
```

Comment:
What happens if you just have a Jinja template in the input file
```yaml
  Rotation: {{ Rotation }}
```
and just do
```py
ringdown_params["Rotation"] = evaluated_fot_dict["Rotation"]
```
The resulting input file doesn't need to look super pretty, just needs to parse.

Reply:
Currently the `schedule` function will recurse over lists, scheduling one run for every item in the list (e.g. to do convergence tests where you give it a list of resolutions or outer radii).

The `str` here is not robust enough because it gives Python's string representation of the object, which may or may not coincide with YAML syntax. Try `yaml.safe_dump(obj)`. This will give you the YAML representation of the object. If needed you can give it options, e.g. so a list is returned as `[1, 2]` instead of `- 1\n- 2\n`.